### DISCUSSION TITLE
How can I get git badges?

### DISCUSSION THREAD

**Discussion by Arturs1123** [DenverCoder1/github-readme-streak-stats#648] [Q&A] [answered]
Hello, everyone.
Could you help me for my github?

**Comment by Luis981123** [comment]
How can I help you?

**Reply by Arturs1123** [reply]
Thank you for your reply.
As said, How can I get git badges?

**Reply by Luis981123** [reply, marked as answer]
Please check this link-https://github.com/orgs/community/discussions/28656

**Reply by Arturs1123** [reply]
Thx

**Reply by Tsedexashu08** [reply]
If u mean those achievements badges u can earn them by achieving tasks..
Check out this list: https://github.com/DenverCoder1/github-readme-streak-stats

**Comment by Tsedexashu08** [comment]
If u mean those achievements badges on profiles u can earn them by achieving tasks.
Check out this list https://drknzz.github.io/GitHub-Achievements/
